src/mongocrypt-marking.c

```diff
@@ -1217,16 +1216,15 @@ static bool _fle2_generate_TextSearchTokenSets(_mongocrypt_key_broker_t *kb,
 
             for (; appendCount > 1; appendCount--) {
                 mc_TextSubstringTokenSet_t tset_copy;
```

Comment:
nit - we only need to do the shallow copy once. all the references are the same in memory.

Reply:
done